src/main/java/Events.java

```diff
@@ -0,0 +1,21 @@
+public class Events extends Task {
```

Comment:
Maybe its better to standardize the way all these classes are named? (Deadline and Task are in singular form while Events and ToDos are in plural form) And since we are talking about class in general I believe singular should be better.